puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -73,8 +73,12 @@
   $s3_memory_cache_size = zulipconf('application_server', 's3_memory_cache_size', '1M')
   $s3_disk_cache_size = zulipconf('application_server', 's3_disk_cache_size', '200M')
   $s3_cache_inactive_time = zulipconf('application_server', 's3_cache_inactive_time', '30d')
+  $nginx_resolver_ip = zulipconf('application_server', 'nameserver', resolver_ip())
```

Comment:
This will call `resolver_ip()` even if the config setting is present, right? So it will fail if `resolv.conf` is empty, and the resulting advice to configure `zulip.conf` won’t help.

Reply:
Ah, absolutely.  Fixed, and tested in that configuration.